tests/autoyast/repos.pm

```diff
@@ -25,6 +25,8 @@ sub run {
     # sles12_minimal.xml profile does not install "ip"
     assert_script_run 'ip a || ifstatus all';
     pkcon_quit;
+    # Enable local repositories if needed
+    zypper_call 'mr -el' # Returns 0 even if repos are already enabled
```

Comment:
You've removed import, which we need.

Reply:
> You've removed import

If by import you mean "use version_utils qw(is_sle is_upgrade);" that I had added in previous PR, we do not need it it here anymore.

Reply:
In case the repos are already enabled, it does just this http://amazing.suse.cz/tests/3605#step/repos/7

Reply:
Ah, utils is already imported. I thought `zypper_call` is in some other lib. Sorry for the confusion.